rootfs/Dockerfile

```diff
@@ -56,7 +58,7 @@ RUN curl -sSL https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.p
 
 # install deis/wal-e from the tip of the boto3-upgrade branch
 # NOTE (bacongobbler): we use the commit here so Docker's cache will bust when we make changes upstream
-RUN pip install --disable-pip-version-check --no-cache-dir git+https://github.com/deis/wal-e.git@dfa35c648b1a7b554122312d6c6b592be372e17a
+RUN pip install --disable-pip-version-check --no-cache-dir git+https://github.com/kmala/wal-e.git@148cd08bf85508b24477aa05084bf4b0b3dc2d01
```

Comment:
Before merge, this should point to the `deis/wal-e` repo. For that reason, does this PR depend on https://github.com/deis/wal-e/pull/4?